### DISCUSSION TITLE
SQL query with bindings for integer column giving error

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#4963] [Help] [answered]
Hi,
I updated my app to v1.0.90 and found that SQL queries that has value binding to integer column is giving error:
```
error: invalid input syntax for type integer: "1,2"
```
I figured out that this only happened to queries that uses the `in` expression with multiple values. And example of the query is something like

```
SELECT * FROM customer WHERE id in ({{ customer_ids }});
```
with the `customer_ids = 1,2`

My expectation of the template output would be 
```
SELECT * FROM customer WHERE in (1,2);
```
But the error message seems to indicate that it is now rendering as
```
SELECT * FROM customer WHERE in ('1,2');
```

How can I change this behaviour?

**Comment by mjashanks** [comment, marked as answer]
I have verified this - will look into this. As far as I can see, this makes using `IN` impossible right now. Thank you!

Issue logged: https://github.com/Budibase/budibase/issues/4978

**Reply by micebrain** [reply]
Issue was solved in the latest update v1.0.99
In case anyone bump into this, the query needs to be updated to exclude the rounded bracket i.e. `IN {{ ids }}`
